care/facility/models/patient_consultation.py

```diff
@@ -338,3 +344,14 @@ def has_object_email_discharge_summary_permission(self, request):
 
     def has_object_generate_discharge_summary_permission(self, request):
         return self.has_object_read_permission(request)
+
+
+class ConsultationClinician(models.Model):
```

Comment:
BaseModel? @sainak

Reply:
Question would be, if we need to store the extra info that the BaseModel provides, `external_id, created_date, modified_date` and also soft delete for this table which is just storing the m2m relations. 🤔

Reply:
Its okay without it, we can track the changes in events